.vib/zookeeper/goss/goss.yaml

```diff
@@ -0,0 +1,30 @@
+file:
+  /opt/bitnami/zookeeper:
```

Comment:
Not sure if this test adds value, since we are already checking the files inside of this folder, this is then made redundant. WDYT?